src/data/dailylimits.txt

```diff
@@ -1,6 +1,124 @@
 1
 # Limiter: daily limit
 # Type	Item name	Tracking property	[Max = 1]
+Cast	Acquire Rhinestones	_rhinestonesAcquired
```

Comment:
Sorting is better than not sorting.  Are we pretty much convinced that we want case sensitive sorts in code where a string sort is used because there is no associated numeric?  Just asking for future reference.

Reply:
I just used IntelliJ's sort function - I have no real opinion on that just yet